src/Microsoft.VisualStudio.Composition.Analyzers/VSMEF001PropertyMustHaveSetter.cs

```diff
@@ -14,7 +14,7 @@ namespace Microsoft.VisualStudio.Composition.Analyzers
     /// Creates a diagnostic when `[Import]` is applied to a property with no setter.
     /// </summary>
     [DiagnosticAnalyzer(LanguageNames.CSharp, LanguageNames.VisualBasic)]
-    public class VSMEF001PropertyMustHaveSetter : DiagnosticAnalyzer
+    public sealed class VSMEF001PropertyMustHaveSetter : DiagnosticAnalyzer
```

Comment:
Making this change to just one class deviates this class from the norm for this project.

Aside: sealing classes is way overdone, IMO.

Reply:
Ah, my mistake then. Most of our analyzer projects have many analyzers. But we don't mark any of them as sealed. Unless there's a compelling reason to do so, please revert this change.